src/placement/jump_map.c

```diff
@@ -31,6 +31,14 @@
 #include <daos/pool_map.h>
 #include <isa-l.h>
 
+enum PL_OP_TYPE {
+	PL_PLACE,
+	PL_PLACE_EXTENDED,
+	PL_REBUILD,
+	PL_REINT,
+	PL_ADD,
+};
```

Comment:
It's better to put some comments to explain these ops, that'll be helpful for understanding the code.